C++/test/searching/linear_search.test.cpp

```diff
@@ -0,0 +1,17 @@
+#define LINEAR_SEARCH_TEST
+#define CATCH_CONFIG_MAIN
+
+#include "third_party/catch.hpp"
+#include "source/searching/linear_search.cpp"
+
+TEST_CASE("Base cases", "[linear search]") {
+    REQUIRE(linear_search(1, vector<int>({1})) == 1);
+    REQUIRE(linear_search(1, vector<int>({0})) == -1);
```

Comment:
Also check that the search returns `-1` when the vector is empty (`vector<int>()`).